projects/demo/src/modules/components/line-chart/examples/1/index.ts

```diff
@@ -21,7 +21,8 @@ export default class Example {
         [200, 150],
         [250, 155],
         [300, 190],
-        [350, 90],
+        [350, 0],
+        [400, 0],
```

Comment:
we already have an issue about this https://github.com/taiga-family/taiga-ui/issues/8115

I think we need to add an example when solving this issue rather than just adding a broken example.